Library/Homebrew/formula.rb

```diff
@@ -2219,7 +2219,7 @@ def install; end
   def inreplace(paths, before = nil, after = nil, audit_result = true) # rubocop:disable Style/OptionalBooleanParameter
     super(paths, before, after, audit_result)
   rescue Utils::Inreplace::Error
-    raise BuildError.new(self, "inreplace", paths, nil)
+    raise BuildError.new(self, "inreplace", paths, {})
```

Comment:
Ca the above value still be `nil` after this change?

Reply:
If you mean the value in the hash, yes because `system` also triggers a BuildError